mathesar/tests/api/test_table_settings_api.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+from django.core.cache import cache
+from sqlalchemy import Column, Integer, MetaData
+from sqlalchemy import Table as SATable
+
+from db.tables.operations.select import get_oid_from_table
+from mathesar import models
+
+
+@pytest.fixture
+def schema_name():
+    return 'table_tests'
+
+
+@pytest.fixture
+def schema(create_schema, schema_name):
+    return create_schema(schema_name)
+
+
+@pytest.fixture
+def column_test_table(patent_schema):
+    engine = patent_schema._sa_engine
```

Comment:
Isn't it better to use the engine from the schema, the schema engine does not necessarily need to point to the default test database

Reply:
All fixtures should (and do, since the big refactor) use the default database, unless explicitly specified. See `mathesar/tests/test_multi_db.py` for how multi-database tests are setup. So, unless you're explicitly working with multi-db stuff, your fixtures will always be on the default test database.